stdpopsim/catalog/EscCol/__init__.py

```diff
@@ -40,39 +37,52 @@
     _chromosomes.append(stdpopsim.Chromosome(
         id=name, length=data["length"],
         synonyms=data["synonyms"],
-        # Lapierre et al. (2016) refer to:
-        #  Genomic adaptive mutation rate: 1e-5, Perfeito et al. (2007), and
-        #  Genomic deleterious mutation rate: 2e−4, Kibota and Lynch (1996).
-        mutation_rate=1e-5+2e-4,
+        # Wielgoss et al. 2011, synonymous substitutions over 40,000 generations.
+        mutation_rate=8.9e-11,
         recombination_rate=0.0))
 
 # mean_conversion_rate=8.9e-11 # not implemented yet!
 # mean_conversion_length=542 # not implemented yet!
 
-#: :class:`stdpopsim.Genome` definition for E. Coli.
-# Chromosome length data is based on strain K-12.
-
 _genome = stdpopsim.Genome(
         chromosomes=_chromosomes,
         assembly_name=genome_data.data["assembly_name"],
         assembly_accession=genome_data.data["assembly_accession"],
         mutation_rate_citations=[
-            _perfeito_et_al.because(stdpopsim.CiteReason.MUT_RATE),
-            _kibota_and_lynch.because(stdpopsim.CiteReason.MUT_RATE),
+            _wielgoss_et_al.because(stdpopsim.CiteReason.MUT_RATE),
             ],
         assembly_citations=[
             _blattner_et_al.because(stdpopsim.CiteReason.ASSEMBLY)])
 
-_species = stdpopsim.Species(
+
+# Nasty hack to overwrite ensembl_id property. Monkey patching won't work in
+# this case, as ensembl_id is a property without a setter.
+class Ecoli(stdpopsim.Species):
```

Comment:
Should we have a different way of specifying the ensembl_id? It seems like this stuff might be common,  and it's only used for doing the maintenance stuff, so feel free to change it if it's getting in the way.

Reply:
I changed ensembl_id to be a class attribute rather than a property.